API/Entities/Interfaces/IEntityBase.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace API.Entities.Interfaces;
+
+public interface IEntityBase
```

Comment:
Would this be better as just `IEntity` ?

Reply:
yes